test/code_size/hello_world_wasm2js.js

```diff
@@ -1,64 +1,72 @@
 var c = Module, g, h, k = new TextDecoder("utf8"), l;
 
-function d(b) {
-    this.exports = function(f) {
-        function m(e) {
-            e.set = function(a, n) {
-                this[a] = n;
+function d(e) {
+    this.exports = function(m) {
+        function n(f) {
+            f.set = function(a, p) {
+                this[a] = p;
             };
-            e.get = function(a) {
+            f.get = function(a) {
                 return this[a];
             };
-            return e;
+            return f;
         }
-        return function(e) {
-            var a = new ArrayBuffer(16777216), n = e.a.a;
-            e = m([]);
+        return function(f) {
+            var a = new ArrayBuffer(16908288), p = new Int8Array(a), r = Math.imul, t = f.a.a;
+            f = n([]);
             return {
                 b: Object.create(Object.prototype, {
-                    grow: {},
+                    grow: {
+                        value: function(b) {
+                            var q = a.byteLength / 65536 | 0;
+                            b = q + (b | 0) | 0;
+                            q < b && 65536 > b && (b = new ArrayBuffer(r(b, 65536)), new Int8Array(b).set(p), 
+                            p = new Int8Array(b), a = b);
+                            return q;
+                        }
+                    },
```

Comment:
This is due to this condition: https://github.com/WebAssembly/binaryen/blob/88d6b7c08f3fbf1b510ae630f83ff8d44321b151/src/wasm2js.h#L769-L774. Something needs to be done about it... I am considering falling back to `INITIAL_MEMORY` by default for WASM2JS.